README.md

```diff
@@ -26,339 +26,212 @@
-  execution, dynamic batching and other features provided by Triton.
-
-* Model pipelines using
+under-development progress towards the next release. The current release is 
+version [2.21.0](https://github.com/triton-inference-server/server/tree/r22.04)
+and corresponds to the 22.04 container release on 
+[NVIDIA GPU Cloud (NGC)](https://catalog.ngc.nvidia.com/orgs/nvidia/containers/tritonserver).**
+
+----
+Triton Inference Server is an inference serving software that streamlines AI 
+inferencing. Triton enables teams to deploy any AI model from multiple deep 
+learning and machine learning frameworks, including TensorRT, TensorFlow, PyTorch, 
+ONNX, OpenVINO, Python and more. Triton supports inference across cloud, data center,
+edge and embedded devices on NVIDIA GPUs, x86 and ARM CPU, or AWS Inferentia. 
+Triton delivers optimized performance for many query types, including real time,
+batched, ensembles and audio/video streaming.
+
+Major features include:
+
+- [Multiple deep learning
+  framework support](https://github.com/triton-inference-server/backend)
+- [Multiple machine learning
+  framework support](https://github.com/triton-inference-server/fil_backend)
+- [Concurrent model
+  execution](docs/architecture.md#concurrent-model-execution)
+- [Dynamic batching](docs/model_configuration.md#dynamic-batcher)
+- [Sequence batching](docs/model_configuration.md#sequence-batcher) and 
+  [implicit state management](docs/architecture.md#implicit-state-management)
+- Provides [Extensible
+  backend](https://github.com/triton-inference-server/backend) with a *backend
+  API*
+- Model pipelines using
   [Ensembling](docs/architecture.md#ensemble-models) or [Business
   Logic Scripting
-  (BLS)](https://github.com/triton-inference-server/python_backend#business-logic-scripting).
-  A Triton *ensemble* represents a pipeline of one or more models and
-  the connection of input and output tensors between those
-  models. *BLS* allows a pipeline along with extra business logic to
-  be represented in Python. In both cases a single inference request
-  will trigger the execution of the entire pipeline.
-
-* [HTTP/REST and GRPC inference
+  (BLS)](https://github.com/triton-inference-server/python_backend#business-logic-scripting)
+- [HTTP/REST and GRPC inference
   protocols](docs/inference_protocols.md) based on the community
   developed [KServe
-  protocol](https://github.com/kserve/kserve/tree/master/docs/predict-api/v2).
+  protocol](https://github.com/kserve/kserve/tree/master/docs/predict-api/v2)
+- A [C API](docs/inference_protocols.md#in-process-triton-server-api) and
+  [Java API](docs/inference_protocols.md#java-bindings-for-in-process-triton-server-api)
+  allow Triton to link directly into your application for edge and other in-process use cases
+- [Metrics](docs/metrics.md) indicating GPU utilization, server
+  throughput, and server latency
+
+## Serve a Model in 3 Easy Steps
```

Comment:
can we say "Serve a Model in 3 Easy Steps and Under 2 Minutes"?

Reply:
I would prefer not to put time in there. I don't think that buys us anything. They are very simple instructions now and time is incredibly variable based on the user, network, machine, etc. What if it takes 5 minutes, someone will complain. Let that be TF Serving's thing.

Reply:
It can also take much longer than a few minutes to pull our docker containers if the user has slow network speed, that's out of our control 😅